src/viur/core/email.py

```diff
@@ -375,3 +380,45 @@ def validateQueueEntity(entity: db.Entity):
             ext = attachment["filename"].split(".")[-1].lower()
             if ext not in EmailTransportSendInBlue.allowedExtensions:
                 raise ValueError("The file-extension %s cannot be send using Send in Blue" % ext)
+
+    @PeriodicTask(60 * 60)
+    @staticmethod
+    def check_sib_quota() -> None:
+        """Periodically checks the remaining SendInBlue email quota.
+
+        This task does not have to be enabled.
+        It automatically checks if the apiKey is configured.
+
+        There are three thresholds: 1000, 500, 100
+        An email will be sent for the lowest threshold that has been undercut.
+        """
+        if conf.get("viur.email.sendInBlue.apiKey") is None:
+            return  # no SIB key, we cannot check
+
+        req = requests.get(
+            "https://api.sendinblue.com/v3/account",
+            headers={"api-key": conf["viur.email.sendInBlue.apiKey"]},
+        )
+        if not req.ok:
+            logging.error("Failed to fetch SIB account information")
+            return
+        data = req.json()
+        logging.debug(f"SIB account data: {data}")
+        for plan in data["plan"]:
+            if plan["type"] == "payAsYouGo":
+                credits = plan["credits"]
+                break
+        else:
+            credits = -1
+
+        logging.info(f"SIB E-Mail credits: {credits}")
+
+        for idx, limit in list(enumerate((1000, 500, 100), 1))[::-1]:
```

Comment:
Maybe we can make this configurable like. 
```suggestion
        for idx, limit in list(enumerate(conf["viur.email.sendInBlue.thresholds"], 1))[::-1]:
```

Reply:
This probably makes sense. Depending on how high the email volume of the website other levels are necessary.